src/main/java/me/kermx/desirepaths/DesirePaths.java

```diff
@@ -179,126 +190,85 @@ private static modifierType getModifier(Player player){
         if (boots == null)
             return modifierType.NO_BOOTS;
         Material bootMaterial = boots.getType();
-        Set<Material> bootMaterials = EnumSet.of(Material.IRON_BOOTS,Material.GOLDEN_BOOTS,Material.DIAMOND_BOOTS,Material.NETHERITE_BOOTS,Material.LEATHER_BOOTS);
-        Optional<Enchantment> featherFallingEnchantment = boots.getEnchantments().keySet().stream().filter(Enchantment.PROTECTION_FALL::equals).findFirst();
-        if (bootMaterials.contains(bootMaterial)){
-            if (featherFallingEnchantment.isPresent()){
+        Set<Material> bootMaterials = EnumSet.of(Material.IRON_BOOTS, Material.GOLDEN_BOOTS, Material.DIAMOND_BOOTS,
+                Material.NETHERITE_BOOTS, Material.LEATHER_BOOTS);
+        Optional<Enchantment> featherFallingEnchantment = boots.getEnchantments().keySet().stream()
+                .filter(Enchantment.PROTECTION_FALL::equals).findFirst();
+        if (bootMaterials.contains(bootMaterial)) {
+            if (featherFallingEnchantment.isPresent()) {
                 return modifierType.FEATHER_FALLING;
             } else {
                 return modifierType.HAS_BOOTS;
             }
-        } else if (bootMaterial == Material.LEATHER_BOOTS){
+        } else if (bootMaterial == Material.LEATHER_BOOTS) {
             return modifierType.LEATHER_BOOTS;
         } else {
             return modifierType.NO_BOOTS;
         }
     }
-    //Handle block at the players feet
```

Comment:
The following section was removed to de-duplicated the handler and switcher methods. They were identical except for the names.